packages/aws-cdk-lib/aws-s3-deployment/test/bucket-deployment.test.ts

```diff
@@ -1709,3 +1709,17 @@ test('DeployTimeSubstitutedFile allows custom role to be supplied', () => {
     },
   });
 });
+
+test('outputObjectKeys to control SourceObjectKeys being sent back', () => {
+  const stack = new cdk.Stack();
+  const bucket = new s3.Bucket(stack, 'Dest');
+  new s3deploy.BucketDeployment(stack, 'Deploy', {
+    sources: [s3deploy.Source.asset(path.join(__dirname, 'my-website'))],
+    destinationBucket: bucket,
+    outputObjectKeys: false,
+  });
+
+  Template.fromStack(stack).hasResourceProperties('Custom::CDKBucketDeployment', {
+    OutputObjectKeys: false,
+  });
```

Comment:
Should we also verify that the previous behaviour still exists as well? That is, have a unit test to check that when `OutputObjectKeys` is true, the object keys are in the output?

Reply:
May also be worth having a test to verify the default value of the attribute

Reply:
added, check integration test cases: packages/@aws-cdk-testing/framework-integ/test/aws-s3-deployment/test/integ.bucket-deployment-big-response.ts && packages/@aws-cdk-testing/framework-integ/test/aws-s3-deployment/test/integ.bucket-deployment.ts